outputs.tf

```diff
@@ -1,3 +1,3 @@
 output "url" {
-  value = "${aws_db_instance.database.engine}://${var.username}:${var.password}@${aws_db_instance.database.endpoint}/${var.name}"
+  value = "${var.engine == "mysql" ? "mysql2" : aws_db_instance.database.engine}://${var.username}:${var.password}@${aws_db_instance.database.endpoint}/${var.name}"
```

Comment:
Does just mysql not work here? I thought it did.

Reply:
It doesn't for Rails when using the `DATABASE_URL` environment variable. It tries to use the `mysql` adapter, which doesn't exist.

Alternatively, we could change this to be `mysql`, keeping it framework agnostic, then update the environment variable for the app by doing a `replace(module.database.url, "mysql://", "mysql2://")`.